libPhoneNumber.xcodeproj/project.pbxproj

```diff
@@ -1344,6 +1344,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO;
```

Comment:
I feel like we should probably fix the quotes in header rather than suppressing the warning, since technically, they're frameworks, so they should have <> instead of ""

Reply:
That was my first approach, was searching to find the root cause.  I'll see what I can do tonight to not suppress the warning and use <> instead of "". Good to know that about frameworks, didn't know that!